template-files/action.yml

```
@@ -0,0 +1,38 @@
+---
+name: Template Files
+description: Template (or copy) files from other repositories and commits them to the specified PR.
+author: Anaconda Inc.
+branding:
+  icon: book-open
+  color: green
+
+inputs:
+  config:
+    description: Configuration path defining what files to template/copy.
+    default: .github/templates/config.yml
+  stubs:
+    description: >-
+      Path to where stub files are located in the current repository.
+    default: .github/templates/
```

This is risky as it may allow loading the config file of the action itself as a template, how about moving that one level deeper?
```suggestion
    default: .github/template-files/templates/
```